src/scripts/components/main/copy-to-clipboard-button/copy-to-clipboard-button.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {FunctionComponent, useEffect, useState} from 'react';
+
+import Button from '../button/button';
+import {CopyTextIcon} from '../icons/copy-text-icon';
+import {CheckIcon} from '../icons/check-icon';
+
+import styles from './copy-to-clipboard-button.module.styl';
+
+interface Props {
+  label: string;
+  handleCopy: () => void;
```

Comment:
not sure how we handled it in this project but I thought we use 'on...' for callbacks, like `onCopy` instead of `handleCopy`